edgexfoundry/device-sdk-go
# feat: Update index files to include names for preloading check

Closes #1843







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/device-sdk-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions
1. run non-secure edgex stack with mqtt message bus
2. build device simple and run `./device-simple -cp -d`
3. stop device simple
4. change the ProfilesDir and ProvisionWatchersDir settings in Consul to use the sample URI files
5. restart device simple
6. ensure that the service starts up properly and if the resource already exists just loads it.

## New Dependency Instructions (If applicable)
